TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TeleOp.java

```diff
@@ -31,6 +32,13 @@ public class TeleOp extends OpMode {
     private final double jointTargetPos = 0.0;
     boolean scoringATM;
 
+    Gamepad currentGamepad1 = new Gamepad();
```

Comment:
could you condense everything into one controller while you're working on teleop anyway? up to you if you want to make a new file or keep it in this one. might want to update portmap too